.appveyor.yml

```diff
@@ -0,0 +1,88 @@
+version: 1.0.{build}
+
+clone_depth: 50
+
+environment:
+  compiler: mingw
+
+  matrix:
+  - target: i686-w64-mingw32
+    compiler_path: mingw32
+    openssl_path: /c/OpenSSL-Win32
+  - target: x86_64-w64-mingw32
+    compiler_path: mingw64
+    openssl_path: /c/OpenSSL-Win64
+
+install:
+- ps: >-
+    Function Execute-Command ($commandPath)
+    {
+        & $commandPath $args 2>&1
+        if ( $LastExitCode -ne 0 ) {
+            $host.SetShouldExit( $LastExitCode )
+        }
+    }
+    Function Execute-Bash ()
+    {
+        Execute-Command 'c:\msys64\usr\bin\bash' '-e' '-c' $args
+    }
+    if ($env:compiler -eq "mingw") {
+            $oldpath = ${env:Path} -split ';'
+            $buildpath = @("C:\msys64\${env:compiler_path}\bin", "C:\msys64\usr\bin") + $oldpath
+            $env:Path = @($buildpath) -join ';'
+            $env:libeventversionfile = @("${env:APPVEYOR_BUILD_FOLDER}", ".appveyor-libevent-version") -join '\'
+            $env:libeventversion = Get-Content "${env:libeventversionfile}" | Out-String
+            Set-Location ".."
+            Execute-Bash "git clone https://github.com/libevent/libevent"
+            Set-Location "libevent"
+            $env:buildlibevent = (Get-Item -Path ".\").FullName
+            Execute-Bash "git checkout -q ${env:libeventversion}"
+            Execute-Bash "./autogen.sh"
+            Execute-Bash "./configure --prefix=/${env:compiler_path} --host=${env:target} --disable-libevent-regress --disable-samples --disable-openssl"
+            Execute-Bash "make -j2 install"

```

Comment:
Do we want to keep building it from scratch? libevent packaged seemed to work previously?

Reply:
I'm confused about what to revert there.  It looks like the commit (a507a62dd637f2b1b7a18bc88c6af402a765e44a) that started building libevent from scratch also removed a bunch of other pacman targets?

Reply:
Yeah it is basically reverting https://github.com/torproject/tor/commit/a507a62dd637f2b1b7a18bc88c6af402a765e44a but fortunately we need only `pacman libevent libevent-dev` without all the OpenSSL and (non-working) zstd stuff.